parquet/src/main/java/org/apache/iceberg/parquet/ParquetWriter.java

```diff
@@ -66,6 +66,9 @@ class ParquetWriter<T> implements FileAppender<T>, Closeable {
   private boolean closed;
   private ParquetFileWriter writer;
   private int rowGroupOrdinal;
+  private long currentBufferSize = 0;
+  private long totalBufferSize = 0;
```

Comment:
Same here, given this is the sum of the raw count of bytes on each add, perhaps something like `totalRawBufferedSize`.